.github/workflows/runners.yaml

```
@@ -25,97 +27,36 @@ on:
       - '!**.md'
 
 env:
+  # Safeguard to prevent pushing images to registeries after build  
+  PUSH_TO_REGISTRIES: false
```

```suggestion
  PUSH_TO_REGISTRIES: true
```

**TODO:** This needs to be committed before merging this PR.